src/supporting-data/citations/components/LiteratureCitation.tsx

```diff
@@ -53,15 +53,24 @@ const Authors: FC<AuthorProps> = ({ authors, limit = 10 }) => {
           <Link to={getLinkToAuthor(author)}>{author}</Link>
         </Fragment>
       ))}
-      {collapsed ? (
+      {hiddenAuthors.length > 0 && (
+        <EllipsisReveal>
+          {', '}
+          {hiddenAuthors.map((author, index) => (
+            // eslint-disable-next-line react/no-array-index-key
+            <Fragment key={index}>
+              {index !== 0 && ', '}
+              <Link to={getLinkToAuthor(author)}>{author}</Link>
+            </Fragment>
+          ))}
+        </EllipsisReveal>
```

Comment:
```suggestion
        <EllipsisReveal>
          {hiddenAuthors.map((author, index) => (
            // eslint-disable-next-line react/no-array-index-key
            <Fragment key={index}>
              , <Link to={getLinkToAuthor(author)}>{author}</Link>
            </Fragment>
          ))}
        </EllipsisReveal>
```

Do double check in a local instance though, I did that in GitHub but unsure if the whitespace will be respected after the ",", otherwhise `{", "}`